apps/bigdl/README.md

```diff
@@ -1,12 +1,14 @@
-# Training VGG network on CIFAR-10 data
+# Training a VGG network on CIFAR-10 Data
 
-> **Disclaimer:** This sample application is provided as-is, without warranty. It is intended to illustrate techniques for implementing various scenarios using Fast Data Platform, but it has not gone through a robust validation process, nor does it use all the techniques commonly employed for highly-resilient, production applications. Please use it with appropriate caution.
+This application is currently only supported on DC/OS. It is being ported to OpenShift and Kubernetes.
+
+> **DISCLAIMER:** This sample application is provided as-is, without warranty. It is intended to illustrate techniques for implementing various scenarios using Fast Data Platform, but it has not gone through a robust validation process, nor does it use all the techniques commonly employed for highly-resilient, production applications. Please use it with appropriate caution.
 
```

Comment:
This was not converted/tested on OS

Reply:
Did you see the first paragraph of this change block?